extensions/amp-wordpress-embed/validator-amp-wordpress-embed.protoascii

```diff
@@ -5,6 +5,7 @@ tags: {  # amp-wordpress-embed
     name: "amp-wordpress-embed"
     version: "1.0"
     version: "latest"
+    bento_supported_version: "1.0"
```

Comment:
This one includes `latest`, since `1.0` is the only version.

Reply:
Done in #35762